### DISCUSSION TITLE
Cloudformation Notifications

### DISCUSSION THREAD

**Discussion by jgallen23** [architect/architect#1245] [Feature Requests]
We'd like to have the team get notified when CloudFormation is finished or errored.  `aws cloudformation deploy` takes a `--notification-arns` parameter that we'd like to be able to set via .arc or env var.  This would let us hit an SNS topic that pings us in slack.

I think it's as simple as adding another argument here: https://github.com/architect/deploy/blob/master/src/sam/01-deploy/index.js#L14

**Comment by ryanblock** [comment]
Neat, I like it. How would you expect it to work?

**Comment by jgallen23** [comment]
a simple approach could look something like:

.arc
```
@app
test

@notifications
{{aws sns arn}}
```

another option would be:

```
arc deploy production --notification-arns [values]
```

**Comment by ryanblock** [comment]
Since this is kind of vendor specific, how about something like:

```
@app
test

@aws
# ... all your other aws stuff
notifications
  {{aws sns arn A}}
  {{aws sns arn B}}
```

**Comment by jgallen23** [comment]
makes sense to me

**Comment by brianleroux** [comment]
I like this feature!

**Comment by brianleroux** [comment]
@jgallen23 just looping back to this… would `.arc` file be easier/better or would you prefer command line ?

**Comment by jgallen23** [comment]
no preference.  Both would work for our workflow (we use github actions to deploy)
